service/auth.go

```diff
@@ -41,7 +41,7 @@ type Auth0Authenticator struct {
 
 // NewAuthenticator returns an Auth0Authenticator that checks for a JWT signed
 // by a key referenced in the JWKS file, has the required audience and issuer
-// claims, and contains claims for a brim tenant and user id.
+// claims, and contains claims for a zui tenant and user id.
```

Comment:
This refers to Brim Data, not the Brim app.
```suggestion
// claims, and contains claims for a brim tenant and user id.
```